validator/client/src/main/java/tech/pegasys/teku/validator/client/DutyScheduler.java

```diff
@@ -31,6 +31,7 @@
   private final DutyLoader epochDutiesScheduler;
   private final StableSubnetSubscriber stableSubnetSubscriber;
   private final NavigableMap<UnsignedLong, DutyQueue> dutiesByEpoch = new TreeMap<>();
+  private volatile UnsignedLong lastAttestationCreationSlot;
```

Comment:
All events coming into this class are on a single thread so this doesn't need to be volatile.